scripts/editor/output-css-variables.js

```diff
@@ -99,18 +99,34 @@ export const outputCssVariables = (attributes, manifest, unique) => {
 
```

Comment:
I couldn't figure out if this part of the code has an option to reset the value to none. It should be something like

```suggestion
if (value === undefined) {
   continue;
}
```

This is quite useful for resetting the values when responsive layout should inherit something from its parent. I could take, as an example wrapper with full width. Let's say that we manually set 
```
{
  large: true,
  desktop: true,
  tablet: false,
  mobile: false,
}
```

but to generate fewer CSS variables this could be set from admin as
```
{
  large: true,
  tablet: false,
}
```
if we set the value to undefined we could achieve this 😄